src/load_cell_endstop.c

```diff
@@ -0,0 +1,297 @@
+// Load Cell based end stops.
+//
+// Copyright (C) 2023  Gareth Farrington <gareth@waves.ky>
```

Comment:
Odd that the copyright is dated 2023 (same comment about the load_cell_probe.py copyright).

Reply:
I think all of these are fixed now